src/app/shared/services/project/organization.service.ts

```diff
@@ -50,6 +50,10 @@ export class OrganizationService {
     getMembersForOrganization(orgID: string) {
         return this.http.get<OrganizationMembersResponse>(`/api/project/organization/${orgID}/members`);
     }
+
+    removeMemberFromOrganization(orgID: string, memberID: string) {
+        return this.http.delete<void>(`/api/project/organization/${orgID}/members/${memberID}`, {});
```

Comment:
```suggestion
        return this.http.delete<void>(`/api/project/organization/${orgID}/members/${memberID}`);
```
The second parameter is optional and therefore not needed.